conans/model/info.py

```diff
@@ -170,12 +178,12 @@ def deserialize(data):
 class ConanInfo(object):
 
     def copy(self):
```

Comment:
Not sure about a `copy` function that only copies the fields that matter to compute the SHA. Call ConanInfo.create()?